### PR TITLE
doc enhancements

### PR DESCRIPTION
**Pull request by shenda1**
# Description
PR is raised to address doc enhancements.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 80% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: templates/guides/import_volumes_mapped_to_sdc.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Importing volumes mapped to SDC using sdc_volumes_mapping resource
+
+This guide explains how to import volumes mapped to SDC using sdc_volumes_mapping_resource. Below steps are taken from [this article](https://developer.hashicorp.com/terraform/language/import/generating-configuration).
+
+### Step 1: Add the import block
+```
```
Comment (rounak-adhikary) [inline templates/guides/import_volumes_mapped_to_sdc.md:29]:
```suggestion
```terraform
```

File: templates/guides/import_volumes_mapped_to_sdc.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Importing volumes mapped to SDC using sdc_volumes_mapping resource
```
Comment (rounak-adhikary) [inline templates/guides/import_volumes_mapped_to_sdc.md:24]:
```suggestion
# Importing existing volume mappings of an SDC using sdc_volumes_mapping resource
```
As we are not importing any volumes. Same goes for the title of this page.

**Review by rounak-adhikary — DISMISSED** [review]

**Review by gokul-srivathsan — APPROVED** [review]
lgtm

**Review by gokul-srivathsan — APPROVED** [review]
LGTM

**Review by Krunal-Thakkar — APPROVED** [review]

**Pull request merged**